### PR TITLE
[DO NOT MERGE UNTIL AGENT IS RELEASED, ETA 12/21] Java agent release 8.8.0

### PR DESCRIPTION
**Pull request by deleonenriqueta**
Added Release Notes for the Java Agent v8.8.0
made some updates to reflect new compatibility and configurations

### CODE REVIEW

**Comment by obenkenobi** [comment]
The goal here is to merge this before 2pm PST

File: src/content/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file.mdx
Comment (homelessbirds) [inline src/content/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file.mdx:713]: @deleonenriqueta I don't think it's a deterrent from pushing this live but what is the workaround? We should link it somewhere.

File: src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-880.mdx
Comment (homelessbirds) [inline src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-880.mdx:27]:
```suggestion
* For upcoming changes to our Synthetics Monitoring, we've added an additional Synthetics Header to identify the synthetics type, initiator and additional attributes for synthetics requests.  [1607](https://github.com/newrelic/newrelic-java-agent/pull/1607)
```

File: src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-880.mdx
Comment (homelessbirds) [inline src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-880.mdx:137]:
```suggestion
We add new settings to `newrelic.yml` as we release new versions of the agent. You can use `diff` or another diffing utility to see what's changed, and add the new config settings to your old file. Make sure not to overwrite any customizations you've made to the file, such as your license key, app name, or changes to default settings.

For example, if you `diff` the default `newrelic.yml` files for Java agent versions 7.10.0 and 7.11.0, the results printed to the console will be like this:
```

File: src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-880.mdx
Comment (homelessbirds) [inline src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-880.mdx:77]:
```suggestion
* Fix transaction naming in Spring controllers with a CGLIB proxy. Transactions now use the actual class name as opposed to the proxied class name.  [1574](https://github.com/newrelic/newrelic-java-agent/pull/1575)
* Fix a `NullPointerException` caused by ServletContext in servlet instrumentation modules. [1636](https://github.com/newrelic/newrelic-java-agent/pull/1636)
* Fix a memory leak caused by Lettuce instrumentation. Duplicate code for transaction linking has been removed from the Lettuce instrumentation and is handled by netty-reactor instead. [1608](https://github.com/newrelic/newrelic-java-agent/pull/1608)
* Fix a bug where invalidating a license key causes a memory leak. Reconnection tasks are now capped in the event of a `LicenseException`.   [1606](https://github.com/newrelic/newrelic-java-agent/pull/1606)
* Fix a `NullPointerException` caused by RPMServiceManager [1604](https://github.com/newrelic/newrelic-java-agent/pull/1604)
* Add a workaround for a memory leak that may occur in rare scenarios with instrumentation using the legacy async API in the Java Agent (which async servlets and Jetty Continuations use). [1555](https://github.com/newrelic/newrelic-java-agent/pull/1555)

The option can be configured via the following means:
```

**Review by homelessbirds — APPROVED** [review]

**Comment by homelessbirds** [comment]
@jeff-colucci I approved this but I think it needs a second review in the morning and a follow-up.

File: src/content/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file.mdx
Comment (obenkenobi) [reply src/content/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file.mdx:713]: @homelessbirds @deleonenriqueta Maybe reword "A configurable workaround has been introduced to prevent such a scenario from occurring." to instead be "This configuration acts as a workaround to prevent such a scenario from occurring."

Comment (deleonenriqueta) [reply src/content/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file.mdx]: Updated the descriptor with the suggested rewording. @homelessbirds @obenkenobi

**Comment by obenkenobi** [comment]
It is uncertain if we can release today so as stated before, this PR should still not be merged.

**Comment by obenkenobi** [comment]
@homelessbirds Just an update, our release is scheduled for 12/21

**Review by jeff-colucci — APPROVED** [review]
Good to go!

**Pull request merged**
